doc/neps/nep-0050-scalar-promotion.rst

```diff
@@ -214,7 +214,7 @@ arrays that are not 0-D, such as ``array([2])``.
      - ``int64(301)``
      - *Exception* [T5]_
    * - ``uint8(100) + 200``
-     - ``int64(301)``
+     - ``int64(300)``
```

Comment:
I think we should go the other way to be consistent with the rows around this one and change `uint8(100)` to `uint8(1)` without changing the result

Reply:
No, sorry, that already is in the table in the row above this. Nevermind, looks good thanks.